code/__DEFINES/lighting.dm

```diff
@@ -59,11 +59,13 @@
 #define LIGHT_RANGE_FIRE		3 //How many tiles standard fires glow.
 
 #define LIGHTING_PLANE_ALPHA_VISIBLE 255
-#define LIGHTING_PLANE_ALPHA_NV_TRAIT 250
+#define LIGHTING_PLANE_ALPHA_NV_TRAIT 175
```

Comment:
considering `LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE` sits at 192, this should be 215 at the least. Otherwise, it's all cool for merge.